sdk/eventhubs/azure-messaging-eventhubs/src/main/java/module-info.java

```diff
@@ -6,7 +6,7 @@
     requires transitive com.azure.core.amqp;
 
     requires com.microsoft.azure.qpid.protonj.extensions;
-    requires proton.j;
+    requires org.apache.qpid.proton.j;
 
```

Comment:
This isn't required as it will come in from the azure-core-amqp module

Reply:
Ahh nice! It used to complain without it. But its probably because of the non-existent module name